mautic/mautic
# Merge 3.3.3 into features

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | https://github.com/mautic/mautic/releases/tag/3.3.3
| Related developer documentation PR URL | 
| Issue(s) addressed                     | See https://github.com/mautic/mautic/releases/tag/3.3.3




#### Description:

This PR merges changes from version [3.3.3](https://github.com/mautic/mautic/releases/tag/3.3.3) to the features branch.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. All the PRs from 3.3.3 were already tested separately. We just have to make sure that the merge conflicts did not introduce any issues and that the build is passing.
